client/app/app.translate.js

```diff
@@ -0,0 +1,13 @@
+import localeEN from "./locales/locale-en.json";
+import localeDE from "./locales/locale-de.json";
+
+let AppTranslate = ($translateProvider) => {
+  "ngInject";
+  $translateProvider
+  .translations("en", localeEN)
```

Comment:
Can we use full locale ids with country code too? `en` -> `en-US|en-GB`.

Some things are country based (e.g. US date formats are different to UK ones).

Angular [1] and angular translate support it. And the browser locale might include the country code.

... even if `en-*` is just mapped to `en` initially...

[1] https://docs.angularjs.org/guide/i18n